docs/relational-databases/system-stored-procedures/sp-rename-transact-sql.md

```diff
@@ -238,7 +238,7 @@ EXEC sp_rename 'Person.Person.ContactMail1', 'NewContact','Statistics';
 The following example renames the `c1` column in the `table1` table to `col1`.
 
 > [!NOTE]  
-> This [!INCLUDE[ssazuresynapse](../../includes/ssazuresynapse_md.md)] feature is still in preview for dedicated SQL pools and is currently available only for objects in the **dbo** schema.
+> This [!INCLUDE[ssazuresynapse](../../includes/ssazuresynapse_md.md)] feature is still in preview for dedicated SQL pools and is currently available only for objects in the **dbo** schema.  `sp_rename` is not available in serverless SQL pools.

```

Comment:
Remove this comment, please.

then we can sign off

Reply:
Is sp_rename available in Synapse Analytics serverless SQL Pools? I get error "PROCEDURE 'sp_rename' is not supported." when I try to run this in a serverless SQL pool. [Serverless SQL pool overview](https://learn.microsoft.com/en-us/azure/synapse-analytics/sql/on-demand-workspace-overview) does list that stored procedures are not supported in serverless SQL pool but it also (currently) lists views as not supported either (I've submitted a [pull request](https://github.com/MicrosoftDocs/azure-docs/pull/101640) to address that issue).

At the top of this page it shows:  **Applies to:  SQL Server (all supported versions)  Azure SQL Database  Azure Synapse Analytics**. It should be clear where this applies in **Azure Synapse Analytics** as this only applies to Synapse dedicated SQL pool. Both SQL pools are equally part of **Azure Synapse Analytics**.